client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/Configuration.java

```diff
@@ -56,7 +56,7 @@
    private final TransactionConfiguration transaction;
    private final Features features;
 
-   Configuration(ExecutorFactoryConfiguration asyncExecutorFactory, Supplier<FailoverRequestBalancingStrategy> balancingStrategyFactory, ClassLoader classLoader,
+   public Configuration(ExecutorFactoryConfiguration asyncExecutorFactory, Supplier<FailoverRequestBalancingStrategy> balancingStrategyFactory, ClassLoader classLoader,
```

Comment:
hmmm probably should use the `ConfigurationBuilder` instead of this method directly.

Reply:
all reated to configuration part will be done in another PR